components/rich_text_editor/extensions/link/autolink.js

```diff
@@ -0,0 +1,87 @@
+import {
```

Comment:
This is mostly boilerplate code for the ProseMirror plugin that is able to manipulate the content more than the TipTap extension could on its own.

Reply:
The link extension that TipTap provides has an "autolink" plugin that I did study to learn how these things work. There are some parts that look similar, but I believe they are mostly boilerplate stuff that most plugins would have no matter what. So while I did write this myself I did take inspiration from the other plugin. Do you think it's worth mentioning here? Shouldn't have issues with the [license](https://github.com/ueberdosis/tiptap/blob/main/LICENSE.md) regardless

Reply:
Yeah wasn't really worried about the licensing, just didn't think there would be a lot that was useful to review if it was pretty standard boilerplate.